napari/layers/_markers_layer/model.py

```diff
@@ -23,8 +22,10 @@ class Markers(Layer):
     ----------
     coords : np.ndarray
         Coordinates for each marker.
-    symbol : str
-        Symbol to be used as a marker
+    symbol : str, Symbols
```

Comment:
```suggestion
    symbol : Symbols or {'arrow', 'clobber', 'cross', 'diamond', 'disc',
                         'hbar', 'ring', 'square', 'star', 'tailed_arrow', 
                         'triangle_down', 'triangle_up', 'vbar', 'x'}
```
and remove that information from the description